### ISSUE TITLE
Add change icon for current file command

### ISSUE THREAD

**Issue by cfe84** [gfxholo/iconic#37]
Hey there!

Love the extension, very beautiful.

I think a cool and relatively straight forward addition would be to add a "Change Icon for current file" command, so that we can set a shortcut and make things a bit more keyboard friendly for what is the most used feature for your extension

**Comment by gfxholo** [comment]
You're right, this would be easy! It'd tie nicely into the keyboard support that's already inside the icon dialog. I'll make it gets a command on the next release (so very soon 👁️).
